Lib/test/test_os.py

```diff
@@ -4660,6 +4679,35 @@ def test_fork(self):
         assert_python_ok("-c", code)
         assert_python_ok("-c", code, PYTHONMALLOC="malloc_debug")
 
+    @unittest.skipIf(_testcapi is None, 'needs _testcapi')
+    @unittest.skipUnless(sys.platform in ("linux", "darwin"),
+                         "Only Linux and macOS detect this today.")
+    def test_fork_warns_when_non_python_thread_exists(self):
```

Comment:
is this originated from CPython source code?

Reply:
Yes
```
    @unittest.skipUnless(sys.platform in ("linux", "darwin"),
                         "Only Linux and macOS detect this today.")
    def test_fork_warns_when_non_python_thread_exists(self):
       ...
```

Reply:
Ah, got it. it came from later then 3.11.